File: app/src/main/java/com/kota/TelnetUI/TextView/TelnetTextViewSmall.java
package com.kota.TelnetUI.TextView;

import android.content.Context;
import android.util.AttributeSet;

public class TelnetTextViewSmall extends TelnetTextView {
    public TelnetTextViewSmall(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
        reloadTextSize();
    }

    public TelnetTextViewSmall(Context context, AttributeSet attrs) {
        super(context, attrs);
        reloadTextSize();
    }

    public TelnetTextViewSmall(Context context) {
        super(context);
        reloadTextSize();
    }

    protected void reloadTextSize() {
        setTextModelSize(TEXT_VIEW_SIZE_SMALL);
    }
}

File: app/src/main/java/com/kota/telnetUI/textView/TelnetTextViewSmall.kt
package com.kota.telnetUI.textView

import android.content.Context
import android.util.AttributeSet

class TelnetTextViewSmall : TelnetTextView {
    constructor(context: Context?, attrs: AttributeSet?, defStyle: Int) : super(
        context!!,
        attrs,
        defStyle
    ) {
        reloadTextSize()
    }

    constructor(context: Context?, attrs: AttributeSet?) : super(context!!, attrs) {
        reloadTextSize()
    }

    constructor(context: Context?) : super(context!!) {
        reloadTextSize()
    }

    override fun reloadTextSize() {
        setTextModelSize(TEXT_VIEW_SIZE_SMALL)
    }
}
